File: src/main/java/com/example/climbingproject/model/ClimbingArea.java
package com.example.climbingproject.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ClimbingArea {

    private Integer id;
    private String name;
    private String description;
    private BigDecimal latitude;
    private BigDecimal longitude;
    private String region;
    private OffsetDateTime createdAt;
}

File: src/main/kotlin/com/example/climbingapi/model/ClimbingArea.kt
package com.example.climbingapi.model

import java.math.BigDecimal
import java.time.OffsetDateTime

data class ClimbingArea(
    val id: Int?,
    val name: String?,
    val description: String?,
    val latitude: BigDecimal?,
    val longitude: BigDecimal?,
    val region: String?,
    val createdAt: OffsetDateTime?
)
